spec/services/medical_copays/request_spec.rb

```diff
@@ -64,6 +64,20 @@
     end
   end
 
+  describe '#get' do
```

Comment:
The tests in this file seem to only test the routing and not the controller. Are we testing the controller anywhere?

Reply:
Hey @kpethtel, thanks for taking the time to review 🙂 . This product is pretty new and is not yet in production so we rapidly stood everything up and are following back up to add test before going live in another [PR](https://github.com/department-of-veterans-affairs/vets-api/pull/8546) over here.